### PR TITLE
Updated data.json (#193)

### PR DESCRIPTION
**Pull request by ahammadshawki8**
## What
* Added my GitHub profile's section at the end of the JSON file.

## Why
* I want to showcase my GitHub profile to the open-source community. 🎓
* The picture at the intro is a GIF. So, it is kind of cool. 🔥
* So, please merge this pull request to the main branch. 🙏🏼

## My GitHub profile
@ahammadshawki8

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -364,5 +364,11 @@
     "githubUsername": "makuzaverite",
     "imageUrl": "https://user-images.githubusercontent.com/53856673/102192030-10423b00-3e6f-11eb-8494-a684cebb2ecc.png",
     "issueNumber": 186
+  },
+    {
+    "name": "Ahammad Shawki",
+    "githubUsername": "ahammadshawki8",
+    "imageUrl": "https://user-images.githubusercontent.com/56157819/102689296-26f9e080-4227-11eb-9e75-08c1f2426889.png",
+    "issueNumber": 193
```
Comment (eddiejaoude) [inline data.json:372]: Thank you 👍 . I think the indentation does not match?

Comment (ahammadshawki8) [reply data.json:372]: Ok, I am checking the indentation.

**Comment by ahammadshawki8** [comment]
Now, I think it is ready for merging.

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Comment by eddiejaoude** [comment]
I have resolved the conflicts

**Pull request merged**